ext/standaloneusers/ang/afformEditUserAccount.aff.html

```diff
@@ -6,7 +6,7 @@
     <af-field name="uf_name" />
     <af-field name="is_active" />
     <af-field name="timezone" />
-    <af-field name="language" />
+    <af-field name="language" defn="{help_pre: ts('Set the user interface language of this user. You can also leave it empty to use the default system language.'), input_attrs: {placeholder: ts('Inherit the default system language')}}" />
```

Comment:
The following string is more concise and already in Transifex because it's used elsewhere in core:
```
placeholder: ts('System default language')
```